lib/muser.h

```diff
@@ -188,6 +227,19 @@ typedef struct {
      * Class code.
      */
     lm_pci_hdr_cc_t     cc;
+
+    /*
+     * PCI capabilities. The user needs to only define the ID and size of each
+     * capability. The actual capability is not maintained by libmuser. When a
+     * capability is accessed the appropriate callback function is called.
+     */
+    lm_cap_t        caps[LM_MAX_CAPS];
```

Comment:
I think the alignment got lost, or there's a mix of spaces and tabs (not sure how github displays it).

For struct definitions where there's a lot of comments between elements, I'm not sure if there's value in trying to align everything. Maybe we want to "copy" (adapt) our internal "C" CODING_STYLE to this project.

Reply:
OK will remove alignment for those.